### PR TITLE
chore(assistant): use ai components

### PR DESCRIPTION
**Pull request by nkrantz**
Updating the Paste Assistant components to use the new `AIChatLog` component from `@twilio-paste/ai-chat-log` instead of the `ChatLog` component from `@twilio-paste/chat-log`.

### CODE REVIEW

File: packages/paste-website/src/components/assistant/AssistantComposer.tsx [hunk 1]
```diff
@@ -30,23 +30,25 @@ export const AssistantComposer: React.FC<{ onMessageCreation: (message: string,
...
-      placeholder="Type here..."
-      onChange={handleComposerChange}
-      ref={editorRef}
-    >
-      <ClearEditorPlugin />
-      <SendButtonPlugin onClick={submitMessage} disabled={selectedThread == null} />
-      <EnterKeySubmitPlugin onKeyDown={submitMessage} />
-      <FocusComposerPlugin selectedThread={selectedThread} />
-    </ChatComposer>
+    <ChatComposerContainer variant="contained">
```
Comment (krisantrobus) [inline packages/paste-website/src/components/assistant/AssistantComposer.tsx:33]: Yes new chat composer! 🤟

File: packages/paste-website/src/components/assistant/AssistantComposer.tsx [hunk 2]
```diff
@@ -30,23 +30,25 @@ export const AssistantComposer: React.FC<{ onMessageCreation: (message: string,
...
+            throw error;
+          },
+        }}
+        ariaLabel="Message"
+        placeholder="Type here..."
+        onChange={handleComposerChange}
+        ref={editorRef}
+      >
+        <ClearEditorPlugin />
+        <SendButtonPlugin onClick={submitMessage} disabled={selectedThread == null} />
```
Comment (krisantrobus) [inline packages/paste-website/src/components/assistant/AssistantComposer.tsx:48]: suggestion: Can you use the action group and button like this example in our docs: https://paste.twilio.design/components/chat-composer#chat-composer-contained-variant

The current way has the button inside the text areas and will block text.

File: packages/paste-website/src/components/assistant/AssistantMessage.tsx [hunk 3]
```diff
@@ -1,25 +1,21 @@
...
-import { Logo } from "../../assets/Logo";
 import { formatTimestamp } from "../../utils/formatTimestamp";
 import { AssistantMarkdown } from "./AssistantMarkdown";
 
 export const AssistantMessage: React.FC<{ threadMessage: ThreadMessage }> = ({ threadMessage }) => {
   return (
-    <ChatMessage variant="inbound">
-      <ChatBubble>
+    <AIChatMessage variant="bot">
+      <AIChatMessageAuthor aria-label={`said by paste assistant at ${formatTimestamp(threadMessage.created_at)}`}>
```
Comment (krisantrobus) [inline packages/paste-website/src/components/assistant/AssistantMessage.tsx:11]: Great aria-label!

**Review by krisantrobus — DISMISSED** [review]

File: packages/paste-website/src/components/assistant/AssistantCanvas.tsx [hunk 4]
```diff
@@ -62,29 +60,51 @@ export const AssistantCanvas: React.FC<AssistantCanvasProps> = ({ selectedThread
...
+              fontWeight="fontWeightMedium"
+              textAlign="center"
+            >
               Keep in mind that this is an experimental tool and so the information provided{" "}
-              <strong>may not be entirely accurate</strong>.
-            </ChatBookendItem>
-            <ChatBookendItem>
+              <Text as="span" fontWeight="fontWeightBold">
+                may not be entirely accurate
+              </Text>
```
Comment (serifluous) [inline packages/paste-website/src/components/assistant/AssistantCanvas.tsx:86]: issue: Can you set the size and color of this to match the text around it?

Screenshot 2024-10-31 at 12 38 48 PM

**Review by serifluous — APPROVED** [review]

**Review by krisantrobus — APPROVED** [review]

**Pull request merged**